### ISSUE TITLE
Bug: Right sidebar navigation does not appear to work

### ISSUE THREAD

**Issue by fuglede** [scipy/scipy-sphinx-theme-v2#59]
On documentation pages for individual NumPy routines, e.g. http://generated-numpy-docs-14648-2-sphinx2.surge.sh/reference/generated/numpy.linalg.cholesky.html#numpy.linalg.cholesky, then the right sidebar contains only the name of the routine whose documentation one is currently reading and nothing else. [Previously](https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.cholesky.html) there would be navigation links to other pages as well as a search bar.

If the idea is to remove the navigation links, the sidebar should probably be removed altogether as it currently adds no value.

**Comment by rgommers** [comment]
I agree, removing the right sidebar will solve a bunch of issues. The left sidebar is fine and has the search box too. And "Previous topic" and "Next topic" aren't useful, so no loss there.

**Comment by Shekharrajak** [comment]
Thanks for the comments. My thought for right sidebar was to make user see the headings/ subheading quickly and navigate to that section smoothly. Have a look in [this demo page](http://pr-42-scipy-sphinx-theme-v2.surge.sh/demo/test_optimize.html). 



> sidebar contains only the name of the routine whose documentation one is currently reading and nothing

Yeah! there are pages where we don't have any heading or subheading so right side bar shows nothing. But even if there is no right sidebar then also we will be putting out body conent in the center and right side will be empty. 

WDYT?

**Comment by fuglede** [comment]
I agree that "Previous topic" and "Next topic" aren't the most useful things in the world, but I *think* I've used them for discoverability previously (but certainly recall thinking that they were a bit odd, and as far as discoverability is concerned, a "See also" section works better).

**Comment by Shekharrajak** [comment]
> TOC ends up being split so that part of it is in the left sidebar and part of it is in the right sidebar; seems a bit unusual.

Left sidebar is all about the page and right side bar shows the heading and subheading of a page. I didn't understand how you are saying it is split.

>  I'm not sure I think it works very well since most of the headings end up being split across multiple lines, which I guess you wouldn't normally do in a TOC; are the <code /> blocks needed/useful in that example?

Yes, it is all about how we make headers and subheaders. So it should be taken care while writing the documentation. Right side bar is just taking the heading and displaying half of the content of the header/subheader will not make any sense.

> I also agree that "Previous topic" and "Next topic" aren't the most useful things in the world, although I think I've used them for discoverability previously (but certainly recall thinking that they were a bit odd, and as far as discoverability is concerned, a "See also" section works better).

We have the `next` and `previous` section button at the end of the page (may be we can have at top as well).
`See also` thing can be added during documentation of the API. We should discuss here about the UI/UX and content part issue should be raised in numpy/numpy repo.

**Comment by Shekharrajak** [comment]
> I agree, removing the right sidebar will solve a bunch of issues. The left sidebar is fine and has the search box too. And "Previous topic" and "Next topic" aren't useful, so no loss there.

I think, it is useful when in cases something link this: [c-api/array.html](http://generated-numpy-docs-14648-3-sphinx2.surge.sh/reference/c-api/array.html), but yes I am agree that in some cases we don't have any subheaders in page so in that case right side bar is not required and should be removed.
